### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Kalunge**
## Bookstore: initialize  project with components
in this milestone I:
- Initialised project using React app
- Added React Redux ```(npm install react-redux)```.
- Structured the application files using a "feature folder" approach using ducks pattern for the Redux files. 
 - Added React Router and set two <Route>s and <Link>s for the app's navigation

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import AddBook from './AddBook';
+
+const Books = () => (
+  <div>
+    <AddBook />
+  </div>
+);
```
Comment (henatan99) [inline src/components/Books.js]: - [ ] As per the project's requirement. Currently, you don't have a method that renders the list of Book components that has book object props. Kindly add a Book component that takes a book object as a prop, and render all Book components mapping an array of book objects. Kindly refer back to the [project's requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md) carefully. 

```
Books - the default view Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet) 
Should have a form for adding a book (no functionality yet)
```

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Kalunge , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (Kalunge) [reply src/components/Books.js]: Hey, thank you for the review, however, it seems that the requirements were updated after I had already done the project. I did not have this on my end.

**Review by henatan99 — APPROVED** [review]
Hi @Kalunge  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**